src/applications/representatives/constants/usip.js

```diff
@@ -0,0 +1,14 @@
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+// ```
+// import manifest from 'applications/login/manifest.json';
+// export const PATH = manifest.rootUrl;
+// ```
+// => No cross app imports allowed: representatives importing from login
+export const PATH = '/sign-in';
```

Comment:
The comment here expresses that this value could be derived from its source of truth were it not for the cross-app import prohibition that is part of having isolated builds / continuous deployment (I can't find the documentation that spells this out). But does this comment add too much noise that isn't quite worth it?